go.sum

```diff
@@ -0,0 +1,4 @@
+github.com/galdor/go-cmdline v1.1.0 h1:r9w9qQwgx4Oms9Y+9R1BZBGV+TLa50b8Tkwk8Gs3R28=
```

Comment:
This is not ok ;-) ! We cannot depend on personal projets... unless you have 4k stars and 5 M downloads... and even then...
Either we move the owneship of this project to JT, or we use something more standard, like https://golang.org/pkg/flag/